.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,29 +2,29 @@
 name: Bug report
 about: Create a report to help us improve
 title: ''
-labels: ''
+labels: 'bug'
 assignees: ''
 
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+**Describe the bug:**
+[Add a clear and concise description of what the bug is.]
```

Comment:
In this file there is no space between these lines, as opposed to the other files.